config/schema.error.yaml

```diff
@@ -0,0 +1,567 @@
+    type: critical
+207:
+    step: warm-up
+    title: 'Failed to warm-up some pages'
+    suggestion: ''
+    stage: post-deploy
+    type: critical
+208:
+    step: time-to-firs-byte
+    title: 'Failed to test time to first byte (TTFB)'
+    suggestion: ''
+    stage: post-deploy
+    type: critical
+227:
+    step: clean-cache
+    title: 'Command `/bin/magento cache:flush` failed'
+    suggestion: 'Check the `cloud.log` for more information. Add `VERBOSE_COMMANDS: ''-vvv''` into `.magento.env.yaml` for more detailed command output.'
+    stage: post-deploy
+    type: critical
+243:
+    title: 'Configuration isn''t defined in the `schema.yaml` file'
+    suggestion: 'Configuration isn''t defined in the `./vendor/magento/ece-tools/config/schema.yaml` file. Check that the config variable name is correct, and that it defined.'
+    stage: general
+    type: critical
+244:
+    title: 'Failed to parse the `.magento.env.yaml` file'
+    suggestion: 'The `./.magento.env.yaml` file format is invalid. Use a YAML parser to check the syntax and fix any errors.'
+    stage: general
+    type: critical
+245:
+    title: 'Unable to read the `.magento.env.yaml` file'
+    suggestion: 'Unable to read the `./.magento.env.yaml` file. Check file permissions.'
+    stage: general
+    type: critical
+246:
+    title: 'Unable to read the `.schema.yaml` file'
+    suggestion: ''
+    stage: general
+    type: critical
+1001:
+    title: 'File app/etc/config.php does not exist'
+    stage: build
+    step: validate-config
+    type: warning
+1002:
+    title: 'The ./build_options.ini file is no longer supported'
+    stage: build
+    step: validate-config
+    type: warning
+1003:
+    title: 'The modules section is missing from the shared config file'
+    stage: build
+    step: validate-config
+    type: warning
+1004:
+    title: 'Configuration is not compatible with this version of Magento'
+    stage: build
+    step: validate-config
+    type: warning
+1005:
+    title: 'Scd options ignored'
+    stage: build
+    step: validate-config
+    type: warning
+1006:
+    title: 'The configured state is not ideal'
+    stage: build
+    step: validate-config
+    type: warning
+1007:
+    title: 'Baler JS bundling cannot be used'
+    stage: build
+    step: run-baler
+    type: warning
+2001:
+    title: 'Cache is configured for a Redis service that is not available. Configuration will be ignored.'
+    stage: deploy
+    step: 'pre-deploy:cache'
+    type: warning
+2002:
+    title: 'The configured state is not ideal'
+    stage: deploy
+    step: validate-config
+    type: warning
+2003:
+    title: 'The directory nesting level value for error reporting has not been configured'
+    stage: deploy
+    step: validate-config
+    type: warning
+2004:
+    title: 'Invalid file config ./pub/errors/local.xml'
+    stage: deploy
+    step: validate-config
+    type: warning
+2005:
+    title: 'Admin data is ignored during update process'
+    stage: deploy
```

Comment:
We can ad suggestion here to remove this data or at least description why it is ignored

Reply:
added